flux2/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.50.0
+version: 0.60.0
```

Comment:
looks good, please also upgrade the flux version (already tested):
source-controller: v0.9.1 --> 0.15.3
helm-controller: v0.8.2 --> 0.11.1
kustomize-controller: v0.9.3 --> 0.13.2